resources/public/api/conf/2.0/examples/adjustments/foreign_property_accounting_adjustments_submit_consolidated_foreignProperty_request.json

```diff
@@ -1,5 +1,7 @@
 {
-  "foreignProperty": {
+  "foreignProperty": [
+  {
+    "countryCode": "FRA",
     "income": {
```

Comment:
foreignTaxTakenOff will be removed from what is submited in income. Alos can it all be formated please